package.json

```diff
@@ -8,12 +8,14 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "branch": "^0.0.0",
```

Comment:
@Shreets please check this npm module once more, it looks it can cause problems in the future: it was created 8 years ago and had no updates after that, it's version is 0.0.0 (not even 0.0.1), git repo is unavaible github.com/classdojo/branch.js

Reply:
Tha package is not being used anywhere and is not a dependency for another package as well. It must have been installed mistakenly. It has now been uninstalled.